packages/core/content-manager/admin/src/preview/utils/previewScript.ts

```diff
@@ -372,12 +525,13 @@ const previewScript = (shouldRun = true) => {
    * Orchestration
    * ---------------------------------------------------------------------------------------------*/
 
-  setupStegaDecoding().then(() => {
+  setupStegaDOMObserver().then((stegaObserver) => {
     const overlay = createOverlaySystem();
     const highlightManager = createHighlightManager(overlay);
-    const observers = setupObservers(highlightManager);
+    const observers = setupObservers(highlightManager, stegaObserver);
+    const scrollManager = setupScrollManagement(highlightManager);
```

Comment:
Nitpick. I'd settle on a naming convention that is convention createXManager vs setupX. But it's whatever and you can also update this down the road if you agree.

Reply:
I'll merge now because it's hard to juggle with several branches, but I agree. I'll add it to the next PR